src/client/client.jsx

```diff
@@ -1,21 +1,39 @@
 import React from 'react';
 import { hydrate } from 'react-dom';
+import { Provider } from 'react-redux';
 // React Router & App Routes
 import BrowserRouter from 'react-router-dom/BrowserRouter';
 import { renderRoutes } from 'react-router-config';
 import routes from '../shared/routes';
+
+import configureStore from '../shared/store/configureStore';
 // Styles
 import './styles/main.scss';
 
-const AppRouter = () => {
+// Initialize store
+const store = configureStore();
```

Comment:
Is this actually doing anything? Same with `Provider` above — if these are just leftover artifacts it'd be better if we removed them I think.